pkgs/games/deliantra/deliantra-server.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchurl, perlPackages
+, autoconf, perl, gperf, optipng, pngnq, rsync, imagemagick, blitz
+, pkg-config, glib, boost, makeWrapper
+}:
+
+let
+  perl-deps = with perlPackages; [
```

Comment:
This is totally fine but I thought I would let you know about `perl.withPackages` which probably works here as well, just in case you didn't know about it.

Reply:
Thanks for the pointer! I didn't, but I think this looks cleaner for this application.